packages/backend/package.json

```diff
@@ -27,7 +27,6 @@
     "@types/tmp": "^0.2.3",
     "@types/uuid": "^8.3.4",
     "apollo-server": "^3.10.1",
-    "auth": "workspace:../auth",
```

Comment:
This seems like a good thing.
However, for my changes to token introspection, I was generating types from
https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/resource-server.yaml
for use in both `auth` and `backend`.
@mkurapov Do you recommend I generate the types separately in both?
Or should we create a new shared types package? :scream:

Reply:
- https://github.com/interledger/rafiki/issues/844

Reply:
I like it, especially that the Open API schema gets defined in a single place. I'm guessing this package will also have `http-signature-utils` as a dependency? Are we planning on signing these AS <> RS introspection requests?

Reply:
We don't do http signatures on token introspection.
In fact, this is unused :point_down: 
https://github.com/interledger/rafiki/blob/1124707caf2f6ed9f7ddae777898973f9107b749/packages/auth/src/config/app.ts#L44

We could include that functionality in the package if we ever wanted it to be useful for setups with different AS/RS operators.